componets/slider/slider.js

```diff
@@ -0,0 +1,7 @@
+var sliderinput = document.querySelector("#sliderinput");
+var sliderOutput = document.querySelector("#sliderOutput");
```

Comment:
You can use `const` here instead of `var`.

Reply:
done